benchmarks/2019.json

```diff
@@ -4841,6 +4895,24 @@
       100
     ]
   },
+  {
+    "measureId": "AAO9",
```

Comment:
Previous:
```
{"level":"debug","message":"measure AAO9 2017 submission method registry single_performance_rate_measurements: 72 measurement-stratas","timestamp":"2019-01-04T00:08:41.854Z"}
{"level":"debug","message":"A total of 57 measurements may be in decile calculation for measure AAO9 registry 4","timestamp":"2019-01-04T00:08:41.854Z"}
{"level":"debug","message":"Success! measure AAO9 has 57 qualified TIN-NPIs reporting for submission method registry","timestamp":"2019-01-04T00:08:41.854Z"}
```

Need to see why this was excluded

Reply:
Removed here:
https://github.com/CMSgov/qpp-measures-data/pull/269/files#diff-19cc2b2b0a838f96141210f2b77688c9L4440

Reply:
Not present in `2019 benchmarks update MAR3` 

keep or remove?